svm/src/transaction_processor.rs

```diff
@@ -336,48 +336,6 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
         }
     }
 
-    /// Returns a hash map of executable program accounts (program accounts that are not writable
```

Comment:
moving the function body like this makes it very difficult to be confident that there were not functional changes (which i'm under the impression there should not be).

prefer separate commits for functionals and cosmetics

Reply:
this entire PR contains only non-functional changes.

Reply:
I moved the function definition back to where it was. Are we good now?